cnf-certification-test/platform/suite.go

```diff
@@ -565,3 +573,20 @@ func testNodeOperatingSystemStatus(env *provider.TestEnvironment) {
 		ginkgo.Fail(b.String())
 	}
 }
+
+func testPodHugePages2M(env *provider.TestEnvironment) {
+	var badPods, errorPods []*provider.Pod
+	for _, put := range env.HugepagesPods {
+		result, err := put.CheckResourceOnly2MiHugePages()
+		if err != nil {
+			tnf.ClaimFilePrintf("Could not check hugepages because of err: %s", err)
+			errorPods = append(errorPods, put)
+			continue
```

Comment:
`CheckResourceOnly2MiHugePages` doesn't return an error, this section might not be needed?

Reply:
fixed